storage-service/src/azureblob.ts

```diff
@@ -0,0 +1,140 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { v4 } from "uuid";
+import config from "./config";
+import { log } from "./index";
+import { FileWithMeta, Metadata, sleep } from "./storage";
+
+const containerName: string = config.azureBlobStorage.containerName;
+
+async function streamToString(readableStream): Promise<string> {
```

Comment:
Can be expressed as

async function streamToString(readableStream): Promise<string> {
  let data = "";
  for await (const chunk of readableStream) {
    data += chunk;
  }
  return data;
}

Reply:
@SamuelPull I believe I copied this from official documentation. Your version is definitely nicer so I will use it.